util/memory/tracker.go

```diff
@@ -150,9 +148,7 @@ func (t *Tracker) Consume(bytes int64) {
 
 // BytesConsumed returns the consumed memory usage value in bytes.
 func (t *Tracker) BytesConsumed() int64 {
-	t.Lock()
-	defer t.Unlock()
-	return t.bytesConsumed
+	return atomic.LoadInt64(&t.bytesConsumed)
```

Comment:
Please also add comments in `Tracker` to make it clear that `Mutex` is only used for protecting `children` field now?

Reply:
Sure, comments have been updated now.